app/views/ansible_credential/_credential_form.html.haml

```diff
@@ -0,0 +1,46 @@
+%br
+.form-horizontal
+  %form#form_div{:name           => "angularForm",
+                 'ng-controller' => "ansibleCredentialsFormController as vm",
+                 'ng-show'       => "vm.afterGet",
+                  "miq-form"     => true,
+                  "model"        => "vm.credentialModel",
+                  "model-copy"   => 'vm.modelCopy',
```

Comment:
I recommend using `ng-clock` here in order to address flickering that is seen sometimes.

`ng-cloak => ""`
(I just tried it in this form and it seems to have fixed it)